ccmlib/scylla_repository.py

```diff
@@ -178,6 +178,8 @@ def extract_version(filename):
 
 
 def get_relocatable_s3_url(branch, s3_version, links):
+    if s3_version.endswith(":debug"):
+        s3_version = s3_version[:-6]
```

Comment:
nit:

```suggestion
    debug_tag = ':debug'
    if s3_version.endswith(debug_tag):
        s3_version = s3_version[:-len(debug_tag)]
```

also python 3.9 and up, has: https://docs.python.org/3.9/library/stdtypes.html?highlight=removeprefix#str.removesuffix
but we'll need first deprecate 3.8 in ccm, python-driver still support 3.8, so it would need ccm to support it as well.